containers/package-builder/checkout-concent-version.sh.j2

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -e
+
+DIRECTORY=$1
+
+# Check if a directory exists
+if [ -d "$DIRECTORY" ]; then
+    git fetch
+else
+    git clone /var/concent-data/concent/ "$DIRECTORY"
+    cd "$DIRECTORY"
+    git fetch
```

Comment:
You do `git fetch` in both cases so you could put it after the loop.

Reply:
Done.